### PR TITLE
Read stale data sample for spanner.

### PR DESCRIPTION
**Pull request by SurferJeffAtGoogle**

### CODE REVIEW

File: spanner/src/read_stale_data.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    $instance = $spanner->instance($instanceId);
+    $database = $instance->database($databaseId);
+    // maxStaleness and minReadTimestamp are only available in single-use transactions.
+    $snapshot = $database->snapshot();
+    $keySet = $spanner->keySet(['all' => true]);
+    $results = $snapshot->read(
+        'Albums',
+        $keySet,
+        ['SingerId', 'AlbumId', 'AlbumTitle'],
+        ['maxStaleness' => new Duration(3)]
```
Comment (vkedia) [inline spanner/src/read_stale_data.php]: can you use exact staleness with 10 seconds. We recommend in our docs to use atleast 10 seconds to get maximum performance benefits. And exact staleness in general would be faster than maxStaleness.

Comment (SurferJeffAtGoogle) [inline spanner/src/read_stale_data.php]: Done.

**Comment by SurferJeffAtGoogle** [comment]
Brent, what error do you see?

**Comment by bshaffer** [comment]
@SurferJeffAtGoogle I'm talking about the error being thrown in travis

**Pull request merged**